src/main/fc/config.c

```diff
@@ -546,6 +546,12 @@ void activateConfig(void)
 
 void validateAndFixConfig(void)
 {
+#if !defined(USE_UNCOMMON_MIXERS) || defined(USE_QUADX_MIXER_ONLY)
+    if (mixers[mixerConfigMutable()->mixerMode].motor == NULL) {
```

Comment:
Not sure if this will work, I think `mixers` is undefined for `USE_QUADX_MIXER_ONLY`.